.github/pull_request_template.md

```diff
@@ -8,6 +8,10 @@ Fixes #[insert GH or internal issue number(s)].
   - [ ] TODO (before merge)
   - [ ] Included
   - [ ] Not necessary because:
+- E2E Tests CI Job required? (Use "e2e" label or ask maintainer to run separately)
+  - [ ] I don't know
```

Comment:
This is useful for external contributors who don't know whether their change requires an e2e run and reminds us, as reviewers, to decide for ourselves.